kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -13525,6 +13593,75 @@ def _set_vf_mac(pci_addr, mac):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = WriteVmHostFileContentResponse()
+        errors = []
+        for hf in cmd.hostFiles:
+            try:
+                if hf.type == 'NvRam':
+                    vm_host_file.write_vm_host_file(hf)
+                elif hf.type == 'TpmState':
+                    tpm.TpmStateHostFile().write_file(hf)
+                else:
+                    errors.append('invalid host file type: %s' % hf.type)
+            except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)